src/components/PopupWithForm/PopupWithForm.js

```diff
@@ -1,42 +1,85 @@
 import './PopupWithForm.css';
+import { useState, useEffect } from 'react';
 
 function PopupWithForm(props) {
   const {
-    isFormOpen,
-    isSignIn,
-    handleCloseModal,
-    toggleLog
+    isLoginFormOpen, closeAllPopups, handleLoginSubmit, handleSignupSubmit
   } = props;
+  const [isLoggingIn, setLoggingIn] = useState(true);
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [username, setUsername] = useState('');
+  const [isInputValid, setInputValid] = useState(true);
+
+  useEffect(() => {
+    setLoggingIn(true);
+  }, [isLoginFormOpen])
+
+  function handleClose() {
+    closeAllPopups();
+  }
+
+  function handleSubmit(event) {
+    isLoggingIn ? handleLoginSubmit(event, email, password) : handleSignupSubmit(event, email, password, username);
+  }
 
   return (
-    <div className={`popup ${isFormOpen && 'popup_active'}`} onClick={(event)=>{event.stopPropagation()}}>
+    <div className={`popup ${isLoginFormOpen && 'popup_active'}`} onClick={(event) => { event.stopPropagation() }}>
       <h2 className="popup__title">
-        {isSignIn ? 'Sign in' : 'Sign up'}
+        {isLoggingIn ? 'Sign in' : 'Sign up'}
       </h2>
-      <form className="popup__form">
+      <form
+        className="popup__form"
+        onSubmit={handleSubmit}
+      >
         <label className='popup__input-label'>
           Email
-          <input className="popup__input" placeholder="Enter email" type='email' required/>
+          <input
+            className="popup__input"
+            placeholder="Enter email" type='email'
+            required
+            onChange={(event) => setEmail(event.target.value)}
+          />
           <p className='popup__error'>error</p>
         </label>
+
         <label className='popup__input-label'>
           Password
-          <input className="popup__input" placeholder="Enter password" type='password' required/>
-          <p className='popup__error'>error</p>
-        </label>
-        <label className={`popup__input-label ${isSignIn && 'popup__input-label_inactive'}`}>
-          Username
-          <input className="popup__input" placeholder="Enter your username" type='text' required/>
+          <input
+            className="popup__input"
+            placeholder="Enter password"
+            type='password'
+            required
+            onChange={(event) => setPassword(event.target.value)}
```

Comment:
If you have `onChange` or ` onInput` in an input then you need to add attribute `value` to this input.